docs/src/static-pages/shadow.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Shadow
```

Comment:
```suggestion
title: Shadows
```

Reply:
I'm going to keep it in singular to match all the other token pages.

Reply:
But this affects the URL too

Reply:
That doesn't matter though IMO. If the URL and text were mismatched before, then this PR fixes that longstanding error

Reply:
Sorry for the delay - was on a call. Yeah tbh it's better to just make them consistent